### ISSUE TITLE
Not sending anything since 1.0.11

### ISSUE THREAD

**Issue by MichelZ** [DarekDan/NLog.Targets.HTTP#25]
We have installed later versions of the target, and it does not seem to send anything anymore. When we switch back to 1.0.10 things work fine.

everything from 11 to 14 does not work for us currently. There seem to be no exceptions/errors in the internal log

**Comment by DarekDan** [comment]
Can you post your config file without Token and actual server name?

**Comment by MichelZ** [comment]
```
public static HTTP GetTarget(string url)
        {
            return new HTTP
            {
                Url = url,
                BatchSize = 100,
                IgnoreSslErrors = true,
                FlushBeforeShutdown = true,
                Layout = new JsonLayout
                {
                    Attributes =
                    {
                        new JsonAttribute("Level", "${level:upperCase=true}"),
                        new JsonAttribute("Source", "${logger:shortName=True}"),
                        new JsonAttribute("Thread", "${event-properties:item=ModuleThreadId}"),
                        new JsonAttribute("Message", "${message}"),
                        new JsonAttribute("Utc", @"${date:universalTime=true:format=yyyy-MM-dd HH\:mm\:ss.fff}"),
                        new JsonAttribute("TenantIdentifier", "${event-properties:item=TenantIdentifier}"),
                        new JsonAttribute("Exception", @"${event-properties:item=ModuleException}"),
                        new JsonAttribute("MappingId", "${event-properties:item=MappingId}"),
                        new JsonAttribute("Mailbox", "${event-properties:item=Mailbox}"),
                        new JsonAttribute("BatchId", "${event-properties:item=BatchId}"),
                        new JsonAttribute("CommandId", "${event-properties:item=CommandId}"),
                        new JsonAttribute("TenantName", "${event-properties:item=TenantName}"),
                        new JsonAttribute("Environment", "${event-properties:item=Environment}"),
                        new JsonAttribute("ModuleName", "${event-properties:item=ModuleName}"),
                        new JsonAttribute("IP", "${mdlc:item=IP}")
                    }
                }
            };
        }
```

**Comment by snakefoot** [comment]
Any output when enabling NLog InternalLogger ? https://github.com/NLog/NLog/wiki/Internal-Logging

**Comment by DarekDan** [comment]
I can confirm that if there is only one message sent to the logger, it will not be written, for reasons yet unknown. However, sending two, works. Looking into it.

**Comment by DarekDan** [comment]
It was silly of me to use shared StringBuilder field. It should be resolved in 1.0.15.

**Comment by MichelZ** [comment]
Hi

With the 1.0.16 version, I now get this in the nlog internal log:

```
2021-11-06 07:13:52.8901 Warn Unknown exception occured Exception: System.NullReferenceException: Object reference not set to an instance of an object.
   at NLog.Targets.Http.HTTP.ResetHttpClientIfNeeded()
   at NLog.Targets.Http.HTTP.SendFast(String message)
```

**Comment by snakefoot** [comment]
Think it is caused by `Authorization.Render(LogEventInfo.CreateNullEvent())` where `Authorization` can be null . 

See also fix: https://github.com/DarekDan/NLog.Targets.HTTP/pull/26/commits/a102e9e0f57a50cbfab2f1078787caaeae38a222

**Comment by MichelZ** [comment]
@DarekDan Any chance to get this merged?

**Comment by DarekDan** [comment]
Resolved in 1.0.18
